packages/api/src/jobs/rss/refreshAllFeeds.ts

```diff
@@ -129,15 +132,15 @@ type QueuePriority = 'low' | 'high'
 export const queueRSSRefreshFeedJob = async (
   jobid: string,
   payload: any,
-  options = { priority: 'high' as QueuePriority }
+  options = { priority: 'low' as QueuePriority }
 ): Promise<Job | undefined> => {
   const queue = await getBackendQueue()
   if (!queue) {
     return undefined
   }
-  return queue.add('refresh-feed', payload, {
+  return queue.add(REFRESH_FEED_JOB_NAME, payload, {
     jobId: `${jobid}_${JOB_VERSION}`,
-    priority: options.priority == 'low' ? 10 : 50,
```

Comment:
@jacksonh the priority for `high` is larger than `low` and this means user added rss feeds were processed after cronjob triggered jobs